codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodecBuilder.java

```diff
@@ -165,6 +165,16 @@ public Http2MultiplexCodecBuilder initialHuffmanDecodeCapacity(int initialHuffma
         return super.initialHuffmanDecodeCapacity(initialHuffmanDecodeCapacity);
     }
 
+    @Override
+    public Http2MultiplexCodecBuilder promisedRequestVerifier(Http2PromisedRequestVerifier promisedRequestVerifier) {
```

Comment:
FYI: this seems like something that would be removed later once the multiplex/frame handlers are split apart, because you could use a handler to intercept push promises without a built-in API.

Reply:
I will revert this part of the change as it isn't required for this PR.